src/main/java/uk/gov/digital/ho/hocs/casework/api/DashboardService.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.digital.ho.hocs.casework.api;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.casework.api.utils.DashboardStatisticFactory;
+import uk.gov.digital.ho.hocs.casework.application.RequestData;
+import uk.gov.digital.ho.hocs.casework.domain.model.*;
+import uk.gov.digital.ho.hocs.casework.domain.repository.StatisticRepository;
+import uk.gov.digital.ho.hocs.casework.security.UserPermissionsService;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static uk.gov.digital.ho.hocs.casework.api.utils.DashboardStatisticFactory.DashboardStatisticHeaders.*;
+import static uk.gov.digital.ho.hocs.casework.application.LogEvent.*;
+
+@Slf4j
+@Service
+public class DashboardService {
+
+    private final DashboardStatisticFactory dashboardStatisticFactory;
+    private final RequestData requestData;
+    private final StatisticRepository statisticRepository;
+    private final UserPermissionsService userPermissionsService;
+
+    public DashboardService(DashboardStatisticFactory dashboardStatisticFactory,
+                            RequestData requestData,
+                            StatisticRepository statisticRepository,
+                            UserPermissionsService userPermissionsService) {
+        this.dashboardStatisticFactory = dashboardStatisticFactory;
+        this.requestData = requestData;
+        this.statisticRepository = statisticRepository;
+        this.userPermissionsService = userPermissionsService;
+    }
+
+    Map<UUID, Map<String, Integer>> getDashboard() {
+        log.debug("Getting dashboard statistics for user");
+        Set<UUID> allUserteams = userPermissionsService.getUserTeams();
+        if (allUserteams.isEmpty()) {
+            log.warn("No teams - Returning 0 Stages", value(EVENT, TEAMS_STAGE_LIST_EMPTY));
```

Comment:
Can we log the relevant user (unless we already do?)

Reply:
Comes through to Kibana through the `X-Auth-UserId` header.